### ISSUE TITLE
Fixed Timezone for UI for all Users (Settings)

### ISSUE THREAD

**Issue by sktnetwork** [jhuckaby/Cronicle#565]
## Summary

I can't find any settings, or options, or config-variable, to set a defined fixed TimeZone for the App/UI.

## Steps to reproduce the problem

N/A.

## Your Setup

I have multiple users (dev team) in different countries, but all my servers and target apps, are on ONE single timezone.
When I see the logs sitting in another country, the logs are always in my timezone, and not the server or event timezone.
All I need is just one timezone to set, so every user, sitting wherever sees the logs in same TZ, and we all can communicate easily with each other, to tell them, e.g. that the event failed at 03:05. Right now when we do, they all need to convert the TZ on their own, because Cronicle picks the TZ based on Browser. 

Good for default, but can we please have a quick setting, so you remove this in app.js:

`this.tz = jstz.determine().name();`

and add it as a setting, if there's a setting in UI ? Or maybe, just in the config, like there's a config for `job_env`, we can have a config for `app_env_tz` or something. Please.

### Operating system and version?

Ubuntu 20.x

### Node.js version?

Latest.

### Cronicle software version?

Version 0.9.11

### Are you using a multi-server setup, or just a single server?
Singh

### Are you using the filesystem as back-end storage, or S3/Couchbase?

Filesystem

### Can you reproduce the crash consistently?
N/A
## Log Excerpts
N/A

**Comment by jhuckaby** [comment]
This is resolved in Cronicle v2 (Orchestra), due for release in 2023.

I'll add a quick config setting for you tho.  Shouldn't take long.

**Comment by jhuckaby** [comment]
I don't think this change is going to do what you want, exactly.  The `app.tz` property is only used to augment the display of dates/times in certain specific situations in the app.

Cronicle v1 is only designed to display dates/times in the current user's timezone.  Allowing you to override the `app.tz` isn't going to solve your problem, alas.

**Comment by sktnetwork** [comment]
Thank you, for taking this up so quickly. I understand the situation, and will wait for Cronicle v2.
